extensions/expanded-opcodes/README.md

```diff
@@ -0,0 +1,87 @@
+| `111 00 000`  | `SS CCCCCC`  | `RRR RRR MM` | Register-Register operation                                               |
+| `111 00 ???`  | `SS CCCCCC`  | `RRR RRR MM` | When `???` is not `000`, reserved                                         |
+| `111 01 000`  | `SS CCCCCC`  | `RRR IIIII`  | Register-Immediate operation                                              |
+| `111 01 ???`  | `SS CCCCCC`  | `RRR IIIII`  | When `???` is not `000`, reserved                                         |
+| `111 10 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative jump                                                       |
+| `111 11 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative call if stack extension is implemented, otherwise reserved |
+
+| Symbol | Meaning                                    |
+|--------|--------------------------------------------|
+| C      | condition code bit                         |
+| R      | register id                                |
+| I      | immediate                                  |
+| D      | signed displacement                        |
+| S      | size                                       |
+| M      | memory mode                                |
+| ?      | reserved for extensions or another purpose |
+
+## Mapping from the base ISA opcodes to this extension
+
+When the base opcode `CCCC` matches `0xxx`, the new opcode is `0 CCCC 0`
+
+When the base opcode `CCCC` matches `1xxx`, the new opcode is `01 CCCC`
+
+## Opcode mapping table
+
+| `CCCCCC` | NAME            | Operation                          | Flags  | Comment     |
+|----------|-----------------|------------------------------------|--------|-------------|
+| `000000` | `ADD`           | `A ← A + B`                        | `ZNCV` |             |
+| `000001` | `ADC`           | `A ← A + B + C`                    | `ZNCV` |             |
+| `000010` | `SUB`           | `A ← A - B`                        | `ZNCV` |             |
+| `000011` | `SBB`           | `A ← A - B - C`                    | `ZNCV` |             |
+| `000100` | `RSUB`          | `A ← B - A`                        | `ZNCV` | (1)         |
+| `000101` | `RSBB`          | `A ← B - A - C`                    | `ZNCV` |             |
+| `000110` | `CMP`           | `_ ← A - B`                        | `ZNCV` |             |
+| `000111` | `RCMP`          | `_ ← B - A`                        | `ZNCV` |             |
+| `001000` | `OR`            | <code>A ← A &#124; B</code>        | `ZN`   | (5)         |
+| `001001` | `NOR`           | <code>A ← ~(A &#124; B)</code>     | `ZN`   | (5)         |
+| `001010` | `XOR`           | `A ← A ^ B`                        | `ZN`   | (5)         |
+| `001011` | `XNOR`          | `A ← ~(A ^ B)`                     | `ZN`   | (5)         |
+| `001100` | `AND`           | `A ← A & B`                        | `ZN`   | (5)         |
+| `001101` | `NAND`          | `A ← ~(A & B)`                     | `ZN`   | (5)         |
```

Comment:
I'm not convinced that NOR, XNOR, or NAND should have opcodes (and certainly not in this extension, regardless). Other ISAs don't have them.

Reply:
It really depends on what bitwise operators we want to have. I just kind of placed them there just because.

Reply:
I removed them since the expanded opcodes section is much more lean in terms of added instructions now